packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: image_picker
 description: Flutter plugin for selecting images from the Android and iOS image
   library, and taking new pictures with the camera.
 homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker
-version: 0.6.7+7
+version: 0.6.8
```

Comment:
Since it is just a documentation update, we should only update the patch version, so 0.6.7+8
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
Changed version number to 0.6.7+8 (missed the part which explained the semantic versioning prior version 1.0.0).